### ISSUE TITLE
Drag and drop: incorrect sound logic

### ISSUE THREAD

**Issue by chayaml** [ji-devs/ji-cloud#3265]
In the current drag & drop state, the the incorrect sound plays either when the user drags and drops the item in the wrong area _or_ when simply clicking on the item. If audio is attached to the item, and the item is being clicked, the incorrect sound will play first, followed by the recorded audio, which becomes confusing because of the flow (the audio is played while the item is dragged and incorrect/ correct sounds plays when item is dropped). I think it would make more sense if the incorrect sound plays when the item is dragged and dropped in the wrong place, but not when the item is clicked. 
@corinnewo

**Comment by corinnewo** [comment]
@chayaml Good catch! I agree that the 'flying' sound should not play on click at all, only the voiceover. The 'flying' sound should only play when the object has been dragged and placed in the wrong place. Please assign to @johnnynotsolucky as I cannot assign at the moment! THanks
